enterprise/frontend/src/embedding-sdk/conventional-changelog-config.js

```diff
@@ -1,4 +1,51 @@
 // eslint-disable-next-line import/no-commonjs
 module.exports = {
-  gitRawCommitsOpts: { grep: "(sdk)" },
+  gitRawCommitsOpts: {
+    grep: "(sdk)",
+  },
+  options: {
+    preset: "angular",
+    pkg: {
+      path: "enterprise/frontend/src/embedding-sdk/package.template.json",
+    },
+    tagPrefix: "embedding-sdk-",
+    transform: function (commit, cb) {
+      if (typeof commit.gitTags === "string") {
+        const tagPrefix = "embedding-sdk-";
+        const rtag = new RegExp(`tag:\\s*[=]?${tagPrefix}(.+?)[,)]`, "gi");
```

Comment:
This was [defined outside the transform function](https://github.com/conventional-changelog/conventional-changelog/blob/65550d9a30cfab4b00b3e982e42bd68fedafcefc/packages/conventional-changelog-core/lib/merge-config.js#L76) in the source code, so I declare it here.

And since we provide `tagPrefix` I don't need to include this check `options && options.tagPrefix`